index.md

```diff
@@ -961,6 +961,45 @@ Command Examples:
       LIST >3                         ; Command to list all channels with
                                       more than three users.
 
+### KICK message
+
+          Command: KICK
+       Parameters: <channel> *( "," <channel> ) <user> *( "," <user> ) [<comment>]
+
+The KICK command can be used to request the forced removal of a user from a channel.
+It causes the `<user>` to [`PART`](#part-message) from the `<channel>` by force.
+If a "comment" is given, this will be sent instead of the default message, the nickname of the user issuing the KICK.
+
+For the message to be syntactically correct, there MUST be either one channel parameter and multiple user parameter, or as many channel parameters as there are user parameters.  
+
+The server MUST NOT send KICK messages with multiple channels or users to clients.
+This is necessarily to maintain backward compatibility with existing client software.
+
+Servers MAY limit the number of target users per `KICK` command via the [`TARGMAX` parameter of `RPL_ISUPPORT`](#targmax-parameter), and silently drop targets if the number of targets exceeds the limit.
+
+Numeric Replies:
+
+* [`ERR_NEEDMOREPARAMS`](#errneedmoreparams-461) `(461)`
+* [`ERR_NOSUCHCHANNEL`](#errnosuchchannel-403) `(403)`
+* ERR_BADCHANMASK ???
```

Comment:
```suggestion
* [`ERR_BADCHANMASK `](#errbadchanmask-476) `(476)`
```

Reply:
oops, forgot to save this file before committing